notebooks/05.Supervised_Learning-Classification.ipynb

```diff
@@ -71,10 +71,8 @@
    },
    "outputs": [],
    "source": [
-    "plt.scatter(X[y == 0, 0], X[y == 0, 1], \n",
-    "            c='blue', s=40, label='0')\n",
-    "plt.scatter(X[y == 1, 0], X[y == 1, 1], \n",
-    "            c='red', s=40, label='1', marker='s')\n",
+    "plt.scatter(X[y == 0, 0], X[y == 0, 1], s=40, label='0')\n",
```

Comment:
the reason for this might have been that zooming in a lot hides part of the line. Not sure.